### ISSUE TITLE
Question about loss NAN

### ISSUE THREAD

**Issue by heishuini** [inclusionAI/TwinFlow#15]
During the training of SD3.5-Twinflow, I encountered a situation where the loss became NAN. I eventually found that t - tt was zero in that line, which caused an anomaly in the calculation of cof_r with 1.0 / (t - tt). Have you ever encountered this situation?
[https://github.com/inclusionAI/TwinFlow/blob/main/src/methodes/twinflow/twinflow.py#L257](https://github.com/inclusionAI/TwinFlow/blob/main/src/methodes/twinflow/twinflow.py#L257)
```
    @torch.no_grad()
    def get_rcgm_target(
        self,
        rng_state: torch.Tensor,
        model: nn.Module,
        F_th_t: torch.Tensor,
        target: torch.Tensor,
        x_t: torch.Tensor,
        t: torch.Tensor,
        tt: torch.Tensor,
        c: List[torch.Tensor],
        N: int,
    ):
        """
        References:
        - RCGM: https://github.com/LINs-lab/RCGM/blob/main/assets/paper.pdf
        """
        t_m = (t - 0.01).clamp_min(tt)
        x_t = x_t - target * 0.01
        _, _, Ft_tar = self.multi_fwd(rng_state, model, x_t, t_m, tt, c, N)
        mask = t < (tt + 0.01)
        cof_l = torch.where(mask, torch.ones_like(t), 100 * (t - tt))
        cof_r = torch.where(mask, 1 / (t - tt), torch.ones_like(t) * 100)
        Ft_tar = (F_th_t * cof_l - Ft_tar * cof_r) - target
        Ft_tar = F_th_t.data - (Ft_tar).clamp(min=-1.0, max=1.0)
        return Ft_tar
```

**Comment by QAQdev** [comment]
We also fix this in the latest commit
